### ISSUE TITLE
Chat font scaling is incompatible

### ISSUE THREAD

**Issue by Keith94** [TakosThings/Metro-for-Discord#12]
I have my chat font scaling set to 150% but the metro theme (obviously) breaks it; is there a reason it enforces a tiny font size? I commented the offending code and it looks fine at 150% scaling:

```css
.message-group .comment .markup {
  font-size: 13px;

.message-group h2 strong {
  font-size: 15px;

.channel-text-area-default textarea {
  font-size: 15px;
```

**Comment by Keith94** [comment]
In 3.1.0b there's a few issues at 150% scaling:

1. User names and message box text aren't scaled correctly

2. Messages are too far to the left when using compact display

**Comment by TakosThings** [comment]
These should now be fixed in 3.1.1
